src/bin/sage-env

```diff
@@ -251,6 +251,19 @@ if [ -z "${SAGE_ORIG_PATH_SET}" ]; then
     SAGE_ORIG_PATH_SET=True && export SAGE_ORIG_PATH_SET
 fi
 if [ -n "$SAGE_LOCAL" ]; then
+    # https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html
+    MAMBA_ROOT_PREFIX="$SAGE_LOCAL/var/lib/sage/conda"
+    CONDA_PREFIX="$MAMBA_ROOT_PREFIX"
+    if [ -d "$CONDA_PREFIX/conda-meta" ]; then
+        export MAMBA_ROOT_PREFIX
+        export CONDA_PREFIX
+        # We do not use the activation script; we only want bin in PATH, nothing else.
+        export PATH="$CONDA_PREFIX/bin:$PATH"
+        # Unset environment variables that may have come from outside
+        unset CONDA_EXE
+        unset CONDA_PYTHON_EXE
+        unset CONDA_DEFAULT_ENV
```

Comment:
Setting these env variables would interfere with an external conda install which would be providing other packages for example python?

Reply:
Good point; I'll make all settings except for PATH local to the package installation script.

Reply:
Done in 5439fd9